Example.java

```diff
@@ -0,0 +1,13 @@
+public class Example
+ {
+   public static void main(String args[])
+	{
```

Comment:
Looks good. Next time, please document your source code with in-line code comments. 

e.g: 

```
//variable declaration
int num;
```